tests/layer_tests/tensorflow_tests/test_tf_ScatterND.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array, float32_array
+from unit_tests.utils.graph import build_graph, regular_op_with_shaped_data, connect, \
+    shaped_data, connect_front
+
+from common.layer_test_class import check_ir_version
+from common.tf_layer_test_class import CommonTFLayerTest
+from common.utils.tf_utils import permute_nchw_to_nhwc
+
+
+class TestTFScatterND(CommonTFLayerTest):
+    def create_tf_scatternd_placeholder_const_net(self, x_shape, indices, updates, ir_version, use_new_frontend):
+        #
+        #   Create Tensorflow model
+        #
+
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_x_shape = x_shape.copy()
+
+            tf_x_shape = permute_nchw_to_nhwc(tf_x_shape, use_new_frontend)
+
+            x = tf.compat.v1.placeholder(tf.float32, tf_x_shape, 'Input')
+            tf_indices = tf.constant(indices)
```

Comment:
@rkazants is there a use case when `indices` and/or `updates` are placeholders (aka dynamic)?

Reply:
@vurusovs, I think this case must be covered with a separate ticket since we have some obstacles connected to layout. For `Gather` with indices it was implemented in the same way.
@evolosen, please create a ticket probably for 2022.2 release to cover other validation cases

Reply:
done, created task #75515